### ISSUE TITLE
A11y_AzureDataStudio_Notebooks-Create_NonTextContrast: Color contrast ratio of focus border over selected control is 1.3:1 which is less than 3:1.

### ISSUE THREAD

**Issue by PoojaNamde** [microsoft/azuredatastudio#22678]
"[Check out Accessibility Insights!] - Identify accessibility bugs before check-in and make bug fixing faster and easier."

### #GitHub Tags:
#A11yMAS; #A11yWCAG2.1; #A11ySev2; #A11yTCS; #DesktopApp; #SH-AzureDataStudio-Linux-Jul2021; #Azure Data Studio; #WCAG1.4.11; #Linux; #NonTextContrast; 

### Environment Details:​
Application Name: MySQL extension for Azure Data Studio 
Application Version: 1.42.0-insider
Operating System: Ubuntu 20.04.2 LTS
GNOME Version: 3.36.8
ORCA: 42.2-1ubuntu1
Distributor ID: Ubuntu
Description: Ubuntu 22.10
Release: 22.10
Codename: kinetic

### Repro Steps:
1. Launch Azure Data Studio application.
2. TAB to "File" and hit ENTER key. TAB to "New Notebook" and hit ENTER key.
3. TAB to "Cell" dropdown and hit ENTER.
4. Navigate using TAB key and observe that color contrast ratio of focus border over selected control is 1.3:1 which is less than 3:1.

### Actual Result:​
Color contrast ratio of focus border over selected control is 1.3:1 which is less than 3:1.

**Similar issue is also observed in below scenario:
Issue 1: Notebooks - Code cell**
1. Launch Azure Data Studio application.
2. TAB to "File" and hit ENTER key. TAB to "New Notebook" and hit ENTER key.
3. TAB to "Cell" dropdown and hit ENTER. TAB to "Cell Code" and hit ENTER.
4. Navigate using TAB key and observe that color contrast ratio of focus border over selected control is 1.3:1 which is less than 3:1.

### Expected Result:​
Color contrast ratio of focus border over selected control should be greater than or equal to 3:1.

### User Impact:
Low vision users will not be able to see on which control the focus is, and will not be able to select the required list option.

**"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility or the Divisional Driver."**

### Attachment:
![GithubIssue1](https://user-images.githubusercontent.com/90901519/228740749-c047fffb-37c9-4cc2-98f9-bb0315a3bc47.png)

![Github issue 2](https://user-images.githubusercontent.com/90901519/228742672-dd13eceb-52a5-4fdc-ba78-e688c22c924a.png)

**Comment by chlafreniere** [comment]
@corivera do you have an easily accessible Linux machine? I'd love to understand if this color contrast issue is not just notebooks-related.

**Comment by corivera** [comment]
Yeah, I have a linux VM I can use

**Comment by alanrenmsft** [comment]
@corivera did you get a chance to try it on Linux?

**Comment by corivera** [comment]
Not yet. I'll take a look now

**Comment by corivera** [comment]
Looks like it's a general color contrast issue on linux. Other buttons and menu items use the same light gray color

**Comment by alanrenmsft** [comment]
@PoojaNamde finally I was able to reproduce the issue with Ubuntu 20 and it turns out to be an issue with the OS's default theme. Ubuntu 18 doens't have such problem.

Please close the issue as external as there is nothing we can do.


![image](https://github.com/microsoft/azuredatastudio/assets/13777222/27cddb01-e479-447f-a0bc-3658f48fdb94)

**Comment by PoojaNamde** [comment]
Verified this issue in the latest Azure data studio- Insider (Version: 1.45.0 Insider). Tried with all the themes and issue is still able to reproduce as color contrast ratio of focus border over selected control is less than 3:1. Issue needs to be fixed.

**Comment by alanrenmsft** [comment]
@PoojaNamde please read my comments above, it is an issue with Ubuntu 20 as we are using the system provided menu. please open an issue for Ubuntu since there is nothing we can do.

**Comment by PoojaNamde** [comment]
#Regressed:05-25-23; #Closed;

As per the SME input, closing this bug.
